src/serializer/ebml.rs

```diff
@@ -399,6 +407,11 @@ mod tests {
         let mut data = [0u8; 10];
         {
             let gen_res = gen_vint(i)((&mut data[..], 0));
+            if let Err(e) = gen_res {
+                trace!("gen_res is error: {:?}", e);
+                // Do not fail if quickcheck generated data is too large
```

Comment:
Maybe check here that it was actually too large

Reply:
I meant fail the test if `i` is actually not too large but writing it out failed

Reply:
I don't know if it is correct now, I'm checking directly the error to discriminate the possible causes for failures

Reply:
I meant checking the actual `i` value. What if there's a bug and for `i==12` it returns `CustomError(0)` wrongly? :)